test/integration/serverless/run.sh

```diff
@@ -104,7 +112,22 @@ function remove_stack() {
 trap remove_stack EXIT
 
 # deploy the stack
-serverless deploy --stage "${stage}"
+if [ "$ARCHITECTURE" == "amd64" ]; then
+    export DEPLOYMENT_REGION=sa-east-1
+    export DEPLOYMENT_BUCKET=integration-tests-deployment-bucket
+    echo "Deploying stack with $ARCHITECTURE architecture to $DEPLOYMENT_REGION, using the $DEPLOYMENT_BUCKET"
+
+    # we use the architecture name amd64, while AWS requires it to be listed as x86_64
+    export DEPLOYMENT_ARCHITECTURE=x86_64
+    serverless deploy --stage "${stage}"
+else 
+    export DEPLOYMENT_REGION=eu-west-1
+    export DEPLOYMENT_BUCKET=integration-tests-deployment-bucket-arm
```

Comment:
Related, why do we need a different deployment bucket for ARM?

Reply:
I was under the impression that the deployment bucket needed to be in the same region as the stack, is this not true?